operator/v1/types_console.go

```diff
@@ -42,6 +42,9 @@ type ConsoleSpec struct {
 	// If not specified, default route will be used.
 	// +optional
 	Route ConsoleConfigRoute `json:"route"`
+	// plugins defines a list of enabled console plugins.
```

Comment:
Please scratch my comment above if `metadata.name` must be unique across all `ConsolePlugin` custom resources.

Reply:
> I think we should implement an equivalent constraint when installing an operator, so that we don't end up with duplicates in the "enabled plugins" list.

 Yes the logic that prevents duplicate plugin names is in the operator ATM. `removeUnavailablePlugins` removes also dups. Will add it into the function comments.